equinity-historical-data/src/main/resources/application.yaml

```diff
@@ -45,4 +45,21 @@ management:
   endpoints:
     web:
       exposure:
-        include: health,info,metrics
\ No newline at end of file
+        include: health,info,metrics
+
+sentry:
```

Comment:
Is the below config is default one? or updated one based on the requirement from the Business?

Reply:
Most of it is default configuration (excluding dsn, app, etc.). I also tweak the rates, as they really depend on tolerance and traffic. For now, I just want to check and have a good set of tracking logs before we set up the alerts. Then we can adjust. But, the lower the traffic, the higher those rate can be.